src/errors/validation_exception.rs

```diff
@@ -24,7 +24,7 @@ use super::types::{ErrorMode, ErrorType};
 use super::value_exception::PydanticCustomError;
 use super::ValError;
 
-#[pyclass(subclass, extends=PyValueError, module="pydantic_core._pydantic_core")]
+#[pyclass(extends=PyException, module="pydantic_core._pydantic_core")]
```

Comment:
Like I explained on Slack, subclassing a pyclass doesn't really work: you need to override `__new__` instead of `__init__` and whatnot. @tiangolo said he found this out the hard way and is currently no longer subclassing in FastAPI. We should remove the subclassability instead of pretending that it helps with backward compatibility. Or wrap this with a Python class.

Reply:
Sorry missed that, yes I can revert that